utils/build/docker/java/spring-boot/pom.xml

```diff
@@ -89,7 +93,15 @@
 		    <artifactId>bcprov-jdk15on</artifactId>
 		    <version>1.70</version>
 		</dependency>
-
+        <dependency>
```

Comment:
do we really need flyway?

Reply:
Nope, not really, the same can be done with spring, removed!